src/main/java/com/epam/emotionalhelp/controller/QuestionController.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.emotionalhelp.controller;
+
+import com.epam.emotionalhelp.module.Question;
+import com.epam.emotionalhelp.service.QuestionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/questions")
+@RequiredArgsConstructor
+public class QuestionController {
+    private final QuestionService questionService;
+
+    @GetMapping("/findAll")
```

Comment:
```suggestion
    @GetMapping
```
No need to use `/findAll` - we can request just `/v1/questions` to get all the questions.